app/Services/Registrar.php

```diff
@@ -85,7 +78,8 @@ public function create(array $data)
             }
 
             $user->country_code = get_country_code();
-            $user->northstar_id = $this->northstar->register($user);
+
+            $northstar_user = Northstar::createUser($user);
```

Comment:
- [x] `phone` to `mobile`
- [ ] adding `source` to the Northstar user

You can make a `$payload = $user->toArray()` to append the source to (and then submit to Northstar) so that you don't run into issues when running `$user->save()` down below.